executor/mock/standalone/standalone_test.go

```diff
@@ -875,3 +876,28 @@ func testTtyNegative(t *testing.T, jobID string) {
 		t.Fail()
 	}
 }
+
+func testCachedDockerPull(t *testing.T, jobID string) {
+
+	var imgName = "nginx"
+	var imgVersion = "latest"
+	res, err := mock.DockerPull(imgName, imgVersion)
+	if err != nil {
+		t.Errorf("Got error during first docker pull: %+v", err)
```

Comment:
Probably worth using testify:
example: https://github.com/Netflix/titus-executor/blob/master/fslocker/fslocker_test.go#L20

You can even add messages:
https://godoc.org/github.com/stretchr/testify/require#NoErrorf

Reply:
Oh, thank goodness.  The Go built in testing lib is pretty bare bones.